src/yadism/coefficient_functions/heavy/test_g1.py

```diff
@@ -0,0 +1,130 @@
+# -*- coding: utf-8 -*-
+import numpy as np
+from eko.thresholds import ThresholdsAtlas
+
+from yadism import observable_name as on
+from yadism.coefficient_functions.heavy import g1_nc as h_g1_nc 
+from yadism.coefficient_functions.heavy import f2_nc as h_f2_nc
+from yadism.coefficient_functions.heavy import f3_nc as h_f3_nc
+import matplotlib.pyplot as plt
+#todo: compare results to Daniel de Florian figure 4-6 
+class MockCouplingConstants:
```

Comment:
```suggestion
import matplotlib.pyplot as plt


#todo: compare results to Daniel de Florian figure 4-6 
class MockCouplingConstants:
```

Reply:
Not sure, you should try: usually is two empty lines before classes, but I don't remember if comments would be attached (as here) or detached (empty lines in the middle).

If possible, write `# TODO:` instead of `#todo:`, since it is recognized by some editors.

Worst case, you could put after the class docstring.